config/prod/dian-hm-prod-migration-s3.yaml

```diff
@@ -13,14 +13,21 @@ parameters:
   # The resource owner
   OwnerEmail: "it@sagebase.org"
   # (Optional) true for read-write bucket, false (default) for read-only bucket
-  AllowWriteBucket: 'true'
+  AllowWriteBucket: "true"
 
   # Settings have default values but can be overriden. Set the below
   # parameters *only* if you want to override the defaults.
 
+  # THIS CURRENTLY ONLY WORKS IN THE us-east-1 REGION!!!!!
+  # Data transfers within the same region between AWS resources are free.
+  # true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+  # This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
+
   # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
-  #  GrantAccess:
-  #    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+  GrantAccess:
+  # - "arn:aws:iam::325565585839:root"   # Required ARN for a synapse bucket
```

Comment:
isn't this a synapse bucket?  if yes, then you'll want to give synapse access

Reply:
Confirmed, it should be synapse-attached, this is outdated